tools/myst/host/dump.c

```diff
@@ -74,10 +74,6 @@ static int dump_enclave_properties(oe_sgx_enclave_properties_t* properties)
 
     printf("xfrm=%#016" PRIx64 "\n", properties->config.xfrm);
 
-    printf(
-        "num_heap_pages=%" PRIu64 "\n",
-        properties->header.size_settings.num_heap_pages);
```

Comment:
Why don't we print heap pages reserved for OE? It's fixed at 32 pages, right?

Reply:
Okay, I added it back but it should be transparent to the user.